torch/onnx/symbolic_opset9.py

```diff
@@ -1539,6 +1539,10 @@ def log1p(g, self):
     return log(g, add(g, sym_help._if_scalar_type_as(g, torch.ones(1), self), self))
 
 
+def log10(g, self):
+    return div(g, log(g, self), sym_help._if_scalar_type_as(g, torch.ones(1) * 10, self))
```

Comment:
Shouldn't this be `div(log(self), log(10))`?
Does the test pass (after you fix it to actually test torch.log10)?

Reply:
yup nice catch.